afk-daily.sh

```diff
@@ -744,6 +744,55 @@ function teamBounties() {
     verifyRGB 240 1775 d49a61 "Collected/dispatched team bounties." "Failed to collect/dispatch team bounties."
 }
 
+# Fix: Script waits forever in the Arena of Heroes #33
+# Attempts to tap the closest Arena of Heroes opponent. Params: opponent
+function tapClosestOpponent() {
+  # Depending on the opponent number sent as a parameter ($1), this function
+  # would attempt to check if there's an opponent above the one sent.
+  # If there isn't, check the one above that one and so on until one is found.
+  # When found, tap on the opponent and exit function.
+    case $1 in
+        1)
+            # Refresh
+            input tap 815 540
+            wait
+            tapClosestOpponent 5
+            ;;
+        2)
+            getColor 820 700 # Opponent 1
+            if [ "$RGB" == "a7f1b7" ]; then # Looking for the button
+                input tap 820 700
+            else
+                tapClosestOpponent 1
+            fi
+            ;;
+        3)
+            getColor 820 870 # Opponent 2
+            if [ "$RGB" == "2daab4" ] || [ "$RGB" == "aff3c0" ]; then # Looking for the button
```

Comment:
Why do we check for two values here and also on line `787`?

Reply:
> Why do we check for two values here and also on line `787`?

The value change if you are fighting for free or using arena tickets.

```shell
            #                                Free         x1
            #  Opponent 1: 820 700      ->        a7f1b7
            #  Opponent 2: 820 870      ->  2eaab4      aff3c0
            #  Opponent 3: 820 1050     ->        a7f1b7
            #  Opponent 4: 820 1220     ->  2daab4      aff3c0
            #  Opponent 5: 820 1400     ->        aaf2bb
```